portfolio/src/main/webapp/script.js

```diff
@@ -14,10 +14,10 @@
 
 /**
  * Adds a random greeting to the page.
- */
+**/ 
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I am left handed', 'I go to Barnard College in NYC', 'Slow down and enjor the simple pleasures of life', 'Do not count the days, make the days count'];
```

Comment:
You can shorten this with s/go to/attend

Reply:
Done!